src/test/java/com/midaswebserver/midasweb/services/UserServiceImpTest.java

```diff
@@ -0,0 +1,441 @@
+        assertTrue("Returned ticker and set ticker wasn't the same", setList.get(0).getTicker().equals(ticker));
+    }
+
+    @Test
+    public void goodAddSymbolInvalidSymbol1() {
+        final User testUser = new User
+                ("MonkeyKing", "PassW0rd", "一个@阿育王。印度", "861065529988");
+        final String symbol = "UECCC";
+        assertTrue("User wasn't added to database", userService.add(testUser));
+        assertFalse("Input symbol was valid, should have been invalid", userService.addSymbolToUser(testUser, symbol));
+    }
+
+    @Test
+    public void goodAddSymbolInvalidSymbol2() {
+        final User testUser = new User
+                ("CabbageMan", "password", "riblueliany@docomo.ne.jp", "4065556666");
+        final String symbol = "APPLE";
+        assertTrue("User wasn't added to database", userService.add(testUser));
+        assertFalse("Input symbol was valid, should have been invalid", userService.addSymbolToUser(testUser, symbol));
+    }
+
+    @Test
+    public void goodAddSymbolValidSymbol1() {
+        final User testUser = new User
+                ("CabbageMan", "password", "riblueliany@docomo.ne.jp", "4065556666");
+        final String symbol = "AAPL";
+        assertTrue("User wasn't added to database", userService.add(testUser));
+        assertTrue("Input symbol was invalid, should have been valid", userService.addSymbolToUser(testUser, symbol));
+    }
+
+    @Test
+    public void goodAddSymbolValidSymbol2() {
+        final User testUser = new User
+                ("MonkeyKing", "PassW0rd", "一个@阿育王。印度", "861065529988");
+        final String symbol = "UEC";
+        assertTrue("User wasn't added to database", userService.add(testUser));
+        assertTrue("Input symbol was invalid, should have been valid", userService.addSymbolToUser(testUser, symbol));
+    }
+
+    @Test
+    public void goodAddSymbolUserNotExist1() {
+        final String symbol = "UEC";
+        assertFalse("Input symbol was valid, should have been invalid", userService.addSymbolToUser(null, symbol));
+    }
+
+    @Test
+    public void goodAddSymbolUserNotExist2() {
+        final String symbol = "AAPL";
+        assertFalse("Input symbol was valid, should have been invalid", userService.addSymbolToUser(null, symbol));
+    }
+
+    @Test
+    public void goodAddSymbolPreviouslySaved1() {
+        final User testUser = new User
+                ("CabbageMan", "password", "riblueliany@docomo.ne.jp", "4065556666");
+        final String symbol = "AAPL";
+        assertTrue("User wasn't added to database", userService.add(testUser));
+        assertTrue("Input symbol and or user was invalid, should have been valid", userService.addSymbolToUser(testUser, symbol));
+        assertTrue("Input symbol and or user was invalid, second add should have returned true", userService.addSymbolToUser(testUser, symbol));
+
+        List<String> dupedTickers = new ArrayList<>();
```

Comment:
Good call on this test, but instead of creating a list, you could have just updated the count of duplicates found, rather than keeping track of them in a list.
```
int dupeCount = 0;
for (...) {
  if (symbol.equalsIgnoreCase(getTicker()) {
     dupeCount++;
}
```

Also, you are checking for dupedTickers.size() == 1, but what if it was > 1?  Would that also be an error?